### DISCUSSION TITLE
Unable to add basic components to a data table

### DISCUSSION THREAD

**Discussion by Qoyyuum** [Budibase/budibase#6791] [Help] [answered]
Following [this guide](https://docs.budibase.com/docs/table), I wanted to add an image component inside the data table. I clicked on the table, clicked Add Component or the plus symbol next to the screen resolutions sizes options in the top right, clicked "Image" in the basic components, and it kept placing it below the table.

![image](https://user-images.githubusercontent.com/971309/179557367-aefd7f6c-6f32-47d0-84d0-429adc4c0f0f.png)

Am I doing this wrong or is that a bug?

**Comment by melohagan** [comment]
HI @Qoyyuum 

Is that a table block by any chance? You can only add images to table components.
![Screenshot 2022-07-25 at 17 13 11](https://user-images.githubusercontent.com/101575380/180825447-764fe894-946a-45e0-b55c-f1353a878270.png)

You will need to specify a data provider for your tables. https://docs.budibase.com/docs/table

**Reply by Qoyyuum** [reply]
Yes it was a table component.

**Comment by Qoyyuum** [comment, marked as answer]
It seems to be a problem when I have several tabs open with the same
screen. It works after closing all duplicate tabs and refresh the active
one. Thanks.

On Tue, Jul 26, 2022 at 12:13 AM melohagan ***@***.***> wrote:

> HI @Qoyyuum <https://github.com/Qoyyuum>
>
> Is that a table block by any chance? You can only add images to table
> components.
> [image: Screenshot 2022-07-25 at 17 13 11]
> <https://user-images.githubusercontent.com/101575380/180825447-764fe894-946a-45e0-b55c-f1353a878270.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/6791#discussioncomment-3224031>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHNELOLDKOMUCNJGKS75WLVV24KXANCNFSM534233JQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<https://about.me/qoyyuum?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=gmail_api&utm_content=thumb>
Abdul Qoyyuum Haji Abdul Kadir
about.me/qoyyuum
<https://about.me/qoyyuum?promo=email_sig&utm_source=product&utm_medium=email_sig&utm_campaign=gmail_api&utm_content=thumb>
